vendor/dumpstate.te

```diff
@@ -3,3 +3,5 @@ dontaudit dumpstate device:file write;
 dontaudit dumpstate unlabeled:file read;
 allow dumpstate hal_power_default:binder call;
 allow dumpstate hal_light_default:binder call;
+allow dumpstate gpu_device:dir r_dir_perms;
+allow dumpstate gpu_device:dir { search };
```

Comment:
this line is not needed, it's already included by the upper line.

Reply:
Thanks for the comment. I have removed the redundant change and updated the PR.
Please review